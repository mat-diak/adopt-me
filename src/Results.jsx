import Pet from "./Pet.jsx";

const Results = ({ pets }) => {
  return (
    <div className="grid gap-4 grid-cols-1 sm:grid-cols-2 lg:grid-cols-3">
      {!pets.length ? (
        <h1>No pets found</h1>
      ) : (
        pets.map((pet) => {
          return (
            <Pet
              name={pet.name}
              animal={pet.animal}
              breed={pet.breed}
              key={pet.id}
              images={pet.images}
              location={`${pet.city}, ${pet.state}`}
              id={pet.id}
            />
          );
        })
      )}
    </div>
  );
};

export default Results;
