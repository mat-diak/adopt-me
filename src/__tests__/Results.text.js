/**
 * @jest-environment jsdom
 */

import { expect, test } from "@jest/globals";
import { create } from "react-test-renderer";
import Results from "../Results";
