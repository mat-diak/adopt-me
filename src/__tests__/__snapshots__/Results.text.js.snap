// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with no pets 1`] = `
<div>
  <h1>
    No pets found
  </h1>
</div>
`;
